### ISSUE TITLE
zabbix-server Unknown metric pgsql.*

### ISSUE THREAD

**Issue by yanboer** [zabbix/zabbix-docker#1028]
##### SUMMARY

zabbix-server cannot get data from PostgreSQL Database.
but MySQL Database is normal.

##### OS / ENVIRONMENT / Used docker-compose files

`docker-compose.yaml`
```yaml
version: '3'

networks:
  net:

services:
  postgres:
    image: "postgres:14.6"
    ports:
      - "5432:5432"
      - "10051:10051"
      - "10050:10050"
      - "80:80"
      - "8080:8080"
    environment:
      POSTGRES_DB: "zabbix"
      POSTGRES_USER: "zabbix"
      POSTGRES_PASSWORD: "zabbix"
    volumes:
      - ./conf/create.sql:/docker-entrypoint-initdb.d/create.sql
    container_name: postgres
    networks:
      net:
        aliases:
          - postgres

  zabbix-server-pgsql:
    image: "zabbix/zabbix-server-pgsql:alpine-6.2.4"
#    ports:
#      - "10051:10051"
    volumes:
      - ./conf/zabbix_server.conf:/etc/zabbix/zabbix_server.conf
    environment:
      DB_SERVER_HOST: "127.0.0.1"
      DB_SERVER_PORT: "5432"
      POSTGRES_USER: "zabbix"
      POSTGRES_PASSWORD: "zabbix"
    container_name: zabbix-server-pgsql
    network_mode: "service:postgres"
    depends_on:
      - postgres
  #    networks:
  #      net:
  #        aliases:
  #          - zabbix-server-pgsql

  zabbix-agent2:
    image: "zabbix/zabbix-agent2:alpine-6.2.4"
#    ports:
#      - "10050:10050"
#    volumes:
#      - ./conf/postgresql.conf:/etc/zabbix/zabbix_agent2.d/plugins.d/postgresql.conf
    environment:
      ZBX_HOSTNAME: "Zabbix server"
      ZBX_SERVER_HOST: "127.0.0.1"
    container_name: zabbix-agent2
    network_mode: "service:postgres"
    depends_on:
      - zabbix-server-pgsql
#    networks:
#      net:
#        aliases:
#          - zabbix-agent2

  zabbix-web-apache-pgsql:
    image: "zabbix/zabbix-web-apache-pgsql:alpine-6.2.4"
#    ports:
#      - "80:80"
#      - "8080:8080"
    environment:
      ZBX_SERVER_HOST: "127.0.0.1"
      ZBX_SERVER_PORT: "10051"
      DB_SERVER_HOST: "127.0.0.1"
      DB_SERVER_PORT: "5432"
      POSTGRES_USER: "zabbix"
      POSTGRES_PASSWORD: "zabbix"
      POSTGRES_DB: "zabbix"
    container_name: zabbix-web-apache-pgsql
    network_mode: "service:postgres"
    depends_on:
      - zabbix-server-pgsql
#    networks:
#      net:
#        aliases:
#          - zabbix-web-apache-mysql
```

##### CONFIGURATION

For complete information please view: https://github.com/yanboer/zabbix-agen2-usage

##### STEPS TO REPRODUCE

```yaml
git clone https://github.com/yanboer/zabbix-agen2-usage.git
cd zabbix-agen2-usage
docker-compose up
```
add PostgreSQL Database to zabbix, and View zabbix-server logs.

##### EXPECTED RESULTS

zabbix is running and able to fetch PostgreSQL Database data.

##### ACTUAL RESULTS

zabbix is running but cannot fetch PostgreSQL Database data.

```paste below
 
```

some zabbix-server logs:
```
   217:20221116:061436.239 enabling Zabbix agent checks on host "test-pgsql-10": interface became available
   212:20221116:061436.298 item "test-pgsql-10:pgsql.ping["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.ping
   209:20221116:061437.298 item "test-pgsql-10:pgsql.dbstat["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.dbstat
   210:20221116:061438.299 item "test-pgsql-10:pgsql.locks["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.locks
   211:20221116:061439.299 item "test-pgsql-10:pgsql.bgwriter["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.bgwriter
   203:20221116:061440.244 discovery rule "test-pgsql-10:pgsql.db.discovery["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.db.discovery
   209:20221116:061440.300 item "test-pgsql-10:pgsql.replication.lag.b["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.replication.lag.b
   204:20221116:061441.245 discovery rule "test-pgsql-10:pgsql.replication.process.discovery["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.replication.process.discovery
   211:20221116:061441.301 item "test-pgsql-10:pgsql.connections["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.connections
   210:20221116:061442.301 item "test-pgsql-10:pgsql.dbstat.sum["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.dbstat.sum
   209:20221116:061443.303 item "test-pgsql-10:pgsql.replication.count["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.replication.count
   211:20221116:061444.303 item "test-pgsql-10:pgsql.replication.lag.sec["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.replication.lag.sec
   210:20221116:061445.303 item "test-pgsql-10:pgsql.replication.recovery_role["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.replication.recovery_role
   212:20221116:061446.304 item "test-pgsql-10:pgsql.replication.status["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.replication.status
   211:20221116:061447.305 item "test-pgsql-10:pgsql.uptime["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.uptime
   209:20221116:061448.305 item "test-pgsql-10:pgsql.autovacuum.count["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.autovacuum.count
   210:20221116:061449.306 item "test-pgsql-10:pgsql.archive["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.archive
   212:20221116:061450.306 item "test-pgsql-10:pgsql.wal.stat["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.wal.stat
   211:20221116:061452.322 item "test-pgsql-10:pgsql.cache.hit["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Cannot evaluate function: not enough data at "last(//pgsql.dbstat.sum.blks_hit.rate) * 100 / (last(//pgsql.dbstat.sum.blks_hit.rate) + last(//pgsql.dbstat.sum.blks_read.rate))".
   212:20221116:061453.319 item "test-pgsql-10:pgsql.oldest.xid["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.oldest.xid
   212:20221116:061454.321 item "test-pgsql-10:pgsql.replication.process["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}"]" became not supported: Unknown metric pgsql.replication.process
   209:20221116:061455.323 item "test-pgsql-10:pgsql.queries["{$PG.URI}","{$PG.USER}","{$PG.PASSWORD}","{$PG.DATABASE}","{$PG.QUERY_ETIME.MAX.WARN}"]" became not supported: Unknown metric pgsql.queries
```

I want to know what happened, and why I can't get the data from PostgreSQL？
Thanks.

some picture:
postgresql:
![postgresql](https://user-images.githubusercontent.com/48461101/202107055-e1a6896c-61a9-4030-9598-2f286bb2a38c.png)
mysql:
![mysql](https://user-images.githubusercontent.com/48461101/202107187-801d516b-59ef-4373-9978-948ce7cae9c3.png)

**Comment by dotneft** [comment]
It is known problem, will be fixed soon. Please follow: https://support.zabbix.com/browse/ZBX-21886

**Comment by yanboer** [comment]
Thank you very much.

**Comment by jokay** [comment]
@dotneft it does still not work for [zabbix/zabbix-agent2:6.0.12-alpine](https://hub.docker.com/r/zabbix/zabbix-agent2/tags?page=1&name=6.0.12-alpine).

Do we need to build and mount the plugin **zabbix-agent2-plugin-postgresql** manually in the container somehow?

**Comment by jokay** [comment]
Nevermind, were not included in [6.0.12](https://github.com/zabbix/zabbix-docker/blob/6.0.12/Dockerfiles/agent2/alpine/Dockerfile#L33) and will be in [6.0.13](https://github.com/zabbix/zabbix-docker/blob/6.0/Dockerfiles/agent2/alpine/Dockerfile#L37) 👍🏼
